package.json

```diff
@@ -28,11 +28,11 @@
   },
   "dependencies": {
     "passport": "^0.3.2",
-    "underscore": "^1.8.2",
-    "eslint": "^2.5.3",
-    "eslint-config-loopback": "^1.0.0"
+    "underscore": "^1.8.2"
   },
   "devDependencies": {
+    "eslint": "^3.1.1",
```

Comment:
You cannot upgrade to eslint@3 because it [requires node v4 or higher](http://eslint.org/blog/2016/07/eslint-v3.0.0-released#requires-nodejs-4-or-higher). You should move the dependency to `devDependencies`, but you'll probably want to leave the version specifier the same.